Tests/MoyaProviderIntegrationTests.swift

```diff
@@ -255,8 +255,9 @@ class MoyaProviderIntegrationTests: QuickSpec {
                         expect(log).to( contain("Request Headers: [:]") )
                         expect(log).to( contain("HTTP Request Method: GET") )
                         expect(log).to( contain("Response:") )
-                        expect(log).to( contain("{ URL: https://api.github.com/zen } { status code: 200, headers") )
```

Comment:
@sunshinejr This is the test that fails locally for me. The "status code: 200, headers" comes back with capitalized words when I run them locally and on CI they're not capitalized.